src/tests/functional/shared_test_classes/src/subgraph/get_output_before_activation.cpp

```diff
@@ -82,3 +83,92 @@ InferenceEngine::Blob::Ptr OutputBeforeActivation::GenerateInput(const Inference
     return FuncTestUtils::createAndFillBlob(info.getTensorDesc(), 2, -1, 100);
 }
 } // namespace SubgraphTestsDefinitions
+
+namespace ov {
+namespace test {
+std::ostream& operator<<(std::ostream& os, const midOutputType& oType) {
+    switch (oType) {
+    case midOutputType::Sub:
+        return (os << "Sub");
+    case midOutputType::Sum:
+        return (os << "Sum");
+    case midOutputType::Mul:
+        return (os << "Mul");
+    default:
+        return (os << "Unknown");
+    }
+}
+
+std::string OutputBeforeActivationNew::getTestCaseName(const testing::TestParamInfo<outputBeforeActivationParams> &obj) {
+    std::string targetDevice;
+    ov::element::Type netPrecision;
```

Comment:
```suggestion
    ov::element::Type input_type;
```
We don't have network precision since v10 IR.